corehq/apps/builds/management/commands/commcare_build_importer.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import requests
+
+from cStringIO import StringIO
+from jenkinsapi.jenkins import Jenkins
+
+from corehq.apps.builds.models import CommCareBuild
+from django.core.management.base import BaseCommand, CommandError
+
+
+# supress requests library INFO logs
+logging.getLogger("requests").setLevel(logging.WARNING)
```

Comment:
This is a global setting, and will set it for the entire runtime (until changed elsewhere) if this module gets imported for any purpose. Maybe better to set it in [settings.py](https://github.com/dimagi/commcare-hq/blob/42f5cca2ade99b819a496f14c3af3201dd2b563d/settings.py#L847-L895) or possibly below in the `interactive_import` method?